spec/lib/tasks/data_migrations_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "rake"
+
+describe "data migration rake tasks" do
```

Comment:
```suggestion
describe "data migration rake tasks", :postgres do
```
Add the `:postgres` argument to make sure that any data you add to the database is cleaned up afterwards.